### PR TITLE
fix: add diesel migration to update local db

### PR DESCRIPTION
**Pull request by kritikmodi**
## Type of Change

- [x] Bugfix

## Description

This is a part of the attempt to make database even across all environments. This PR contains the migrations for local database.

### Additional Changes

- [x] This PR modifies the database schema

## Motivation and Context

This PR closes https://github.com/juspay/hyperswitch/issues/1809.

Following are the schema comparisons between different environments : 

- Local vs Integ : https://github.com/juspay/hyperswitch-cloud/commit/97d0b001632a5c971f3bd9d959b7d7c7d75e511d
- Local vs Sandbox : https://github.com/juspay/hyperswitch-cloud/commit/2daf82adf0988d2f42dd7a13fbca649eca6349c9
- Local vs Prod : https://github.com/juspay/hyperswitch-cloud/commit/f9a0090acc09d37aed33e62c7005c8f3969149a0

## How did you test it?

Compiles

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed submitted code

### CODE REVIEW

File: migrations/2023-07-28-111829_update_columns_to_fix_db_diff/up.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+ALTER TABLE dispute
+ALTER COLUMN payment_id TYPE VARCHAR(64);
+
+ALTER TABLE merchant_key_store
+ALTER COLUMN created_at SET DEFAULT now();
```
Comment (SanchithHegde) [inline migrations/2023-07-28-111829_update_columns_to_fix_db_diff/up.sql:5]: The defaults were intentionally removed. See #991 and #984. (Also, PR #1043 has a commit that removes these defaults.)

File: migrations/2023-07-28-111829_update_columns_to_fix_db_diff/up.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+ALTER TABLE payment_methods
+ALTER COLUMN payment_method_type TYPE VARCHAR(64);
```
Comment (SanchithHegde) [inline migrations/2023-07-28-111829_update_columns_to_fix_db_diff/up.sql:8]: Will this work all the payment method types we use till date?

**Review by SanchithHegde — COMMENTED** [review]
Other than that, looks good to me!

File: migrations/2023-07-28-111829_update_columns_to_fix_db_diff/up.sql (hunk 1 above)
Comment (kritikmodi) [reply migrations/2023-07-28-111829_update_columns_to_fix_db_diff/up.sql:5]: Okay, removing this.

**Review by inventvenkat — DISMISSED** [review]

**Review by SanchithHegde — APPROVED** [review]

**Review by inventvenkat — APPROVED** [review]

**Review by Narayanbhat166 — APPROVED** [review]

**Review by jarnura — APPROVED** [review]

**Pull request merged**